batcher/aligned-sdk/src/communication/messaging.rs

```diff
@@ -114,6 +114,10 @@ pub async fn send_messages(
                 error!("Invalid replacement message!");
                 return Err(SubmitError::InvalidReplacementMessage);
             }
+            ValidityResponseMessage::InvalidPaymentServiceAddress => {
+                error!("Invalid payment service address!");
+                return Err(SubmitError::InvalidPaymentServiceAddress);
+            }
```

Comment:
for debugging, it may be useful to know which was the expected payment service address and which one the received in the message

Reply:
implemented 👍